src/plugins/intel_cpu/src/nodes/executors/pooling.hpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "cpu_memory.h"
+#include "onednn/iml_type_mapper.h"
+#include "executor.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+struct PoolingAttrs {
+    bool isMaxPool8 = false;
```

Comment:
`isMaxPool8` is not expected to be used on Executors level so lets remove from the structure.

Reply:
Fixed